modules/logging-loki-storage.adoc

```diff
@@ -0,0 +1,274 @@
+// Module is included in the following assemblies:
+// logging/logging-loki-ocp.adoc
+//
+:_content-type: PROCEDURE
+[id="logging-loki-storage_{context}"]
+= Loki log storage
+
+The Loki Operator supports link:https://aws.amazon.com/[AWS S3], link:https://azure.microsoft.com[Azure], link:https://cloud.google.com/[GCS], link:https://min.io/[Minio],  link:https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation[OpenShift Data Foundation], link:https://us.alibabacloud.com/[Alibaba Cloud] and link:https://docs.openstack.org/swift/latest/[Swift] for LokiStack object storage.
+
+The recommended nomenclature for Loki storage is `logging-loki-<type>`. General secret creation format is as follows:
+//Generic secret creation
+include::snippets/logging-create-secret-snip.adoc[]
+
+You can verify a secret has been created by running the following command:
+
+[source,terminal]
+----
+oc get secrets
+----
+
+The table below displays the `type` values within the `LokiStack` CR for each storage provider. For more detailed information, consult the section for your storage provider.
+
+[options="header"]
+.Secret type quick reference
+|================================================
+| Storage provider          | Secret `type` value
+| AWS                       | s3
+| Azure                     | azure
+| Google Cloud              | gcs
+| Minio                     | s3
+| OpenShift Data Foundation | s3
+| Swift                     | swift
+| Alibaba Cloud             | alibabacloud
```

Comment:
loki-operator doesn't support type:alibabacloud,  can we remove alicould from the doc?  https://github.com/openshift/loki/blob/main/operator/bundle/openshift/manifests/loki.grafana.com_lokistacks.yaml#L4285

Reply:
@periklis  Do we plan to add alibabacloud in future(Logging 5.8)?

Reply:
It is already into the code base. I just do not recall if we have a QE approval on this. So let's skip this for now.

Reply:
Already removed. :)